pegasus/sites.v3/hourofcode.com/i18n/en.yml

```diff
@@ -33,6 +33,14 @@
   africa_campaign_date_year: "2020"
   africa_campaign_date_full_year: "October 1-31, 2020"
 
+  hoc2020_header: 'Learn today, build a brighter tomorrow.'
+  hoc2020_header_line1: 'Learn today,'
+  hoc2020_header_line2: 'build a brighter tomorrow.'
```

Comment:
We should be careful to make sure these get translated properly as a single sentence.  It would probably be safer if we translated them as a single string and had something in that string that the code could use to trigger a line break.

Reply:
that's a good point, what could we put in the string to trigger a line break?

Reply:
discussed offline, will do as follow-up PR